modules/databend/src/main/java/org/testcontainers/containers/DatabendContainerProvider.java

```diff
@@ -0,0 +1,26 @@
+package org.testcontainers.containers;
```

Comment:
it should be `org.testcontainers.databend`

Reply:
Sorry omitted it. Fixed.